src/hermes/commands/deposit/invenio.py

```diff
@@ -132,6 +137,90 @@ def deposit(click_ctx: click.Context, ctx: CodeMetaContext):
     _log.info("Published record: %s", record["links"]["record_html"])
 
 
+def _resolve_latest_invenio_id(ctx: CodeMetaContext) -> str:
```

Comment:
This function returns `None` if the last check fails

Reply:
That's correct and intended... I was not quite sure how to eventually handle the error here. It's also related to your comment below

Reply:
Now, a ValueError is raised... needs to be cleaned up further, maybe raise a custom error or so?